From 62d1d443e40e3ce7b8600ba61b41e39701e2430d Mon Sep 17 00:00:00 2001
From: Rai Harninder-B01044 <harninder.rai@freescale.com>
Date: Tue, 27 Oct 2015 13:20:12 +0530
Subject: [PATCH 012/124] arm64: Define ARCH_LAYERSCAPE for Layerscape SoC
 family

Introduce a common ARCH_LAYERSCAPE for layerscape SoC family
and remove the erstwhile specific ARCH_FSL_LS2085A

Signed-off-by: Rai Harninder-B01044 <harninder.rai@freescale.com>
---
 arch/arm64/Kconfig           | 6 +++---
 arch/arm64/configs/defconfig | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index 6f0a3b4..bf4c50b 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -180,10 +180,10 @@ config ARCH_EXYNOS7
 	help
 	  This enables support for Samsung Exynos7 SoC family
 
-config ARCH_FSL_LS2085A
-	bool "Freescale LS2085A SOC"
+config ARCH_LAYERSCAPE
+	bool "Freescale Layerscape SoC family"
 	help
-	  This enables support for Freescale LS2085A SOC.
+	  This enables support for Freescale Layerscape SoC family
 
 config ARCH_MEDIATEK
 	bool "Mediatek MT65xx & MT81xx ARMv8 SoC"
diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 2ed7449..bfdc78c 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -32,7 +32,7 @@ CONFIG_MODULE_UNLOAD=y
 # CONFIG_BLK_DEV_BSG is not set
 # CONFIG_IOSCHED_DEADLINE is not set
 CONFIG_ARCH_EXYNOS7=y
-CONFIG_ARCH_FSL_LS2085A=y
+CONFIG_ARCH_LAYERSCAPE=y
 CONFIG_ARCH_MEDIATEK=y
 CONFIG_ARCH_SEATTLE=y
 CONFIG_ARCH_TEGRA=y
-- 
2.1.0.27.g96db324

